src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -118,6 +118,13 @@ class DPMSolverMultistepScheduler(SchedulerMixin, ConfigMixin):
              This parameter controls whether to use Karras sigmas (Karras et al. (2022) scheme) for step sizes in the
              noise schedule during the sampling process. If True, the sigmas will be determined according to a sequence
              of noise levels {σi} as defined in Equation (5) of the paper https://arxiv.org/pdf/2206.00364.pdf.
+        lambda_min_clipped (`float`, default `-5.1`):
+            the clipping threshold for the minimum value of lambda(t) for numerical stability. This is critical for
+            cosine (squaredcos_cap_v2) noise schedule.
+        is_predicting_variance (`bool`, default `False`):
```

Comment:
Note that in other schedulers such as `DDPM` we have an input called `variance_type`: https://github.com/huggingface/diffusers/blob/efc48da23bd342ca79a79def02649b1975393ea0/src/diffusers/schedulers/scheduling_ddpm.py#L342 

Could we use the same here?
```suggestion
        variance_type (`str`, *optional*):
            Set to "learned" or "learned_range" for diffusion models that predict variance. For example, OpenAI's guided-diffusion
            (https://github.com/openai/guided-diffusion) predicts both mean and variance of the Gaussian distribution
            in the model's output. DPM-Solver only needs the "mean" output because it is based on diffusion ODEs.
```

This way, we're more compatible with:
https://github.com/huggingface/diffusers/blob/efc48da23bd342ca79a79def02649b1975393ea0/src/diffusers/schedulers/scheduling_ddpm.py#L342